.gitignore

```diff
@@ -4,6 +4,7 @@ build
 dist
 *.pyc
 .vscode
+.idea
```

Comment:
is this intended to be added?

Reply:
Yes, it's the configuration directory for IntelliJ based IDE's